### ISSUE TITLE
[FEATURE] change background for quote Reply

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#284]
### Description

the background color for the quote for the two chat type should be different and should be the opposite.

in chat.css do this:
```
.message-block.me blockquote {
        @apply bg-red    
    }
.message-block.other blockquote {
        @apply bg-secondary
    }
```

### Screenshots

receiver:
![image](https://github.com/Dun-sin/Whisper/assets/78784850/38110e65-4d0a-4b38-923e-2662b1702ada)
sender:
![image](https://github.com/Dun-sin/Whisper/assets/78784850/47fa61a4-ff7e-4588-a330-e5a3c019c18b)

### 👀 Have you checked if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Contributing Guidelines?

- [X] I have read and understood the rules in the [Contributing Guidelines](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md)

### Are you willing to work on this issue ?

None

**Comment by harsh-bardolia** [comment]
I would like to work on it.

**Comment by Dun-sin** [comment]
> I would like to work on it.

Definitely, thanks for wanting to contribute, Make sure you follow the rules [👉🏽here👈🏽](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted and will be closed. Good Luck

**Comment by harsh-bardolia** [comment]
# Fixes Issue 
## My PR closes #284

## 👨‍💻 Changes proposed(What did you do ?)
- change background for quote Reply

## ✔️ Check List (Check all the applicable boxes)
-[] Solve the Blockquoto background, for the receiver & sender.
-[] Made both components with different colors.

# Note to reviewers:
## 📷 Screenshots

- ## For Receiver:
![image](https://github.com/Dun-sin/Whisper/assets/110019311/1e4d462a-06fa-4cba-a74c-0d3051a4d4e9)


- ## For Sender:
![image](https://github.com/Dun-sin/Whisper/assets/110019311/30997d57-9e77-45d8-ace6-1b889c8ac2d9)

**Comment by Dun-sin** [comment]
> I would like to work on this issue, is it possible.

don't think, so it's already assigned to someone else in case you missed that but if they give up on it, it will be passed to you, in the meantime, you can check other issues that aren't assigned to someone or use the app and create issues on problems/suggestions you find, thanks for wanting to contribute
